examples/fuzzing/rt_n12/fuzz.py

```diff
@@ -0,0 +1,173 @@
+from qiling import *
+from qiling.const import *
+from qiling.extensions import afl
+from typing import Optional
+from qiling.os.const import *
+from qiling.extensions.coverage import utils as cov_utils
+import argparse
+
+payload_location = 0 # location in snapshot to be modified
```

Comment:
Global variables are not encouraged in general, but if you still need them please try to minimize their scope: it looks like `fuzz` function is the right scope for it.

Reply:
>  it looks like fuzz function is the right scope for it.

Moved.